### PR TITLE
fix: #197 Webpack does not load nested dependencies

### PR DESCRIPTION
**Pull request by dobromir-hristov**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] It's been tested on Windows
- [ ] It's been tested on Linux
- [ ] It's been tested on MacOS
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar-framework.org/tree/dev/source) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Fixes an issue where Webpack was not loading dependencies that were nested inside package's node_modules reported in #197 

**Details:**
When two packages share a common package, but with different versions, one of the packages gets a nested `node_modules` holding the required version of the package it depends on.

The lack of `node_modules` string means that Webpack will only do a shallow look for packages inside the specified absolute paths.

The change is tested locally and is checked against resources like the Vue CLI, where they do the same.

### CODE REVIEW

File: lib/webpack/create-chain.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = function (cfg, configName) {
     fileHash = needsHash ? '.[hash:8]' : '',
     chunkHash = needsHash ? '.[contenthash:8]' : '',
     resolveModules = [
+      'node_modules',
```
Comment (rstoenescu) [inline lib/webpack/create-chain.js]: Hi, isn't this same as "appPaths.resolve.app('node_modules')" ?

**Comment by dobromir-hristov** [comment]
No. If provided with an absolute path, like we do now, it only looks for shallow level deps. It requires that string to actually look deeply into dependencies.On 26 Oct 2018 11:59, Razvan Stoenescu <notifications@github.com> wrote:@rstoenescu commented on this pull request.



In lib/webpack/create-chain.js:
> @@ -16,6 +16,7 @@ module.exports = function (cfg, configName) {
     fileHash = needsHash ? '.[hash:8]' : '',
     chunkHash = needsHash ? '.[contenthash:8]' : '',
     resolveModules = [
+      'node_modules',

Hi, isn't this same as "appPaths.resolve.app('node_modules')" ?

File: lib/webpack/create-chain.js (hunk 1 above)
Comment (dobromir-hristov) [reply lib/webpack/create-chain.js]: https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/config/base.js#L43-L46
Vue CLI also has the same config :)

**Comment by rstoenescu** [comment]
Hi Dobromir,

Sorry for taking so long. Currently focusing on v1.0, so please bare with me a little more until I properly review, merge and publish. Thanks for this PR.

**Comment by rstoenescu** [comment]
Thanks for investigating and providing a PR!

**Comment by dobromir-hristov** [comment]
;) Always bud.

**Pull request merged**